app/src/main/res/values/strings.xml

```diff
@@ -90,6 +90,8 @@
     <string name="show_search_suggestions_summary">Show suggestions when searching</string>
     <string name="enable_search_history_title">Search history</string>
     <string name="enable_search_history_summary">Store search queries locally</string>
+    <string name="unlimited_search_history_title">Unlimited search history</string>
+    <string name="unlimited_search_history_summary">Unlimited search history entries</string>
```

Comment:
A better suited description would probably be "Show as many search suggestions as possible from history"

Reply:
Does this show all suggestions or only the related ones? Maybe "Display all (related) entries from search history" does also fit.